tempesta_fw/http.c

```diff
@@ -59,6 +59,9 @@ int ghprio; /* GFSM hook priority. */
 #define S_H_CONN_KA		S_F_CONNECTION S_V_CONN_KA S_CRLFCRLF
 #define S_H_CONN_CLOSE		S_F_CONNECTION S_V_CONN_CLOSE S_CRLFCRLF
 
+static void tfw_http_resp_forward(TfwHttpReq *req,
+				  TfwHttpResp *resp, bool unref_data);
```

Comment:
Please avoid forward function declarations. If you think that there are program modules in the compilation unit, so you arrange functions according to the modules, then it seems the compilation unit should be separated to the program modules.